src/Microsoft.OData.Core/Json/TextWriterWrapper.cs

```diff
@@ -76,6 +77,30 @@ public override void Flush()
             this.writer.Flush();
         }
 
+        /// <summary>
+        /// Increases the level of indentation applied to the output asynchronously.
+        /// </summary>
+        public virtual async Task IncreaseIndentationAsync()
+        {
+            await TaskUtils.CompletedTask.ConfigureAwait(false);

```

Comment:
>.ConfigureAwait(false) [](start = 41, length = 22)

Shouldn't need to call ConfigureAwait on a completed task (recurring comment)

Reply:
@mikepizzo Initially I had this method without the `async` modifier as follows:
```
public virtual Task IncreaseIdentationAsync()
{
    return TaskUtils.CompletedTask;
}
```
Acting on review feedback, I added `async` and then added `.ConfigureAwait(false)` to pacify the analyzer. In your opinion, do you think its okay to ditch the `async` modifier and just return the completed task? Or do I just suppress the analyzer warning?

Reply:
`ConfigureAwait` on an already completed task has no effect. That said I'd stick with `async` and suppress the analyzer

Reply:
I would keep the async but supress the warning (as you've done).

---
In reply to: [582566983](https://github.com/OData/odata.net/pull/2002#discussion_r582566983) [](ancestors = 582566983)

Reply:
:thumbsup: 

---
In reply to: [582591007](https://github.com/OData/odata.net/pull/2002#discussion_r582591007) [](ancestors = 582591007)